vendor/revisions.json

```diff
@@ -1,4 +1,8 @@
 {
+  "v17": [
+    "17.0",
+    "9156d63ce253bed9d1f76355ceec610e444eaffa"
```

Comment:
Will you bump the submodule to https://github.com/neondatabase/postgres/pull/499 in a separate PR?

Reply:
yes that is the plan